front/src/common/Map/Search/MapSearchLine.tsx

```diff
@@ -63,19 +69,19 @@ const MapSearchLine: React.FC<MapSearchLineProps> = ({
 
   const debouncedSearchTerm = useDebounce(searchState, 300);
 
-  const updateSearch = async (params: searchPayloadType) => {
+  const updateSearch = async (params: SearchPayload) => {
```

Comment:
Rename the parameter searchPayload
```suggestion
  const updateSearch = async (searchPayload: SearchPayload) => {
```
and then l74
```
    const results = await postSearch({ searchPayload }).unwrap();
```